CMakeLists.txt

```diff
@@ -91,6 +91,15 @@ if (FIBER_ENABLE_FFTADVMPI)
 endif()
 
 find_package(MPI REQUIRED)
+if (NOT MPI_FOUND OR (MPI_FOUND AND NOT MPI_ROOT))
+  message(WARNING "MPI package not properly found")
```

Comment:
I would say "not properly configured" (it's "found" alright but not configured as it should be)

Reply:
I do not know if the error comes from the configuration of MPI or the way the find_package works.
So, the least seems to say *not properly found* to me.

Reply:
When `MPI_FOUND` is set then MPI was found. Otherwise it's not found. It's a binary choice. Then you can as questions about the MPI: version, location, etc., which are implementation or configuration issues. And you test those issues in your subsequent code by asking for MPI settings: there is no more looking/finding.

Reply:
Ok, to illustrate what I am saying I copy/paste below what I have on saturn that let the cmake finishes without error but causes error during the compilation:
`CMake Warning (dev) at CMakeLists.txt:93 (find_package):                                                                                                                                                                                                                                  
  Policy CMP0074 is not set: find_package uses <PackageName>_ROOT variables.                                                                                                                                                                                                              
  Run "cmake --help-policy CMP0074" for policy details.  Use the cmake_policy                                                                                                                                                                                                             
  command to set the policy and suppress this warning.                                                                                                                                                                                                                                    
                                                                                                                                                                                                                                                                                          
  Environment variable MPI_ROOT is set to:                                                                                                                                                                                                                                                
                                                                                                                                                                                                                                                                                          
    /home/scayrols/Installs/ompi-git/gcc-7.3.0/gcc-7.3.0/hwloc-2.5.0/cuda-11.6.0/ucx-1.11.2                                                                                                                                                                                               
                                                                                                                                                                                                                                                                                          
  For compatibility, CMake is ignoring the variable.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found MPI_C: /home/scayrols/Installs/ompi-git/gcc-7.3.0/gcc-7.3.0/hwloc-2.5.0/cuda-11.6.0/ucx-1.11.2/lib/libmpi.so (found version "3.1") 
-- Found MPI: TRUE (found version "3.1")`

Obviously MPI is *found* but it is not enough.